CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Change Log
 
+## 11.0.0 (September 17, 2020)
+
+- Enhancement: Support for `cjs/json5` for `prettier`. ([@shrujalshah28](https://github.com/**mvshmakov**) in [#2612](https://github.com/vscode-icons/vscode-icons/pull/2612))
```

Comment:
REmove asterisks

Reply:
The correct link is `https://github.com/shrujalshah28`

Reply:
solved